MON-4057: Expose `scrapeInterval` setting for UWM Prometheus (#2503)
openshift/cluster-monitoring-operator: Documentation/api.md

| Property | Type | Description |
| -------- | ---- | ----------- |

> | scrapeInterval | string | Configures the default interval between consecutive scrapes in case the ServiceMonitor or PodMonitor resource does not specify any value. The interval needs to be greater than or equal to 5 seconds and less than or equal to 5 minutes. The value can be expressed as: * seconds, for example `30s`. * minutees, for example `1m`. * a mix of minutes and seconds, for example `1m30s` The default value is `30s`. |

Reviewer: The formatting is a bit off.
```suggestion
| scrapeInterval | string | Configures the default interval between consecutive scrapes in case the ServiceMonitor or PodMonitor resource does not specify any value. The interval needs to be greater than or equal to 5 seconds and less than or equal to 5 minutes. The value can be expressed as seconds (for example `30s`.), minutes (for example `1m`.) or a mix of minutes and seconds (for example `1m30s`). The default value is `30s`. |
```

> | scrapeInterval | string | Configures the default interval between consecutive scrapes in case the ServiceMonitor or PodMonitor resource does not specify any value. The interval needs to be greater than or equal to 5 seconds and less than or equal to 5 minutes. The value can be expressed as: * seconds, for example `30s`. * minutees, for example `1m`. * a mix of minutes and seconds, for example `1m30s` The default value is `30s`. |

Reviewer 2: ```suggestion
| scrapeInterval | string | Configures the default interval between consecutive scrapes in case the `ServiceMonitor` or `PodMonitor` resource does not specify any value. The interval must be set between 5 seconds and 5 minutes. The value can be expressed in seconds, for example `30s`. * minutees, for example `1m`. * a mix of minutes and seconds, for example `1m30s` The default value is `30s`. |
```

I added some additional suggestions, plus I agree with Simon's suggestions.
